kiali-server/templates/deployment.yaml

```diff
@@ -99,8 +99,8 @@ spec:
             {{- else }}
             scheme: HTTP
             {{- end }}
-          initialDelaySeconds: 5
-          periodSeconds: 30
+          initialDelaySeconds: {{ .Values.server.readiness_probe.initial_delay_seconds | default 5 }}
+          periodSeconds: {{ .Values.server.readiness_probe.period_seconds | default 30 }}
```

Comment:
these are more appropriate under .Values.deployment.
Define the defaults in the values.yaml file since they are being used here (that's why the PR test is red).